src/plugins/url.rs

```diff
@@ -39,51 +39,55 @@ impl UrlPlugin {
     }
 }
 
+impl UrlPlugin {
+    async fn lookup_url_title(&self, ctx: &Context, arg: &str) -> Result<()> {
+        let parsed_url = url::Url::parse(&arg)?;
+
+        let mut buf = String::new();
+
+        // Read in at most 4k of data
```

Comment:
Where's the 4k here? I see 1MiB but not 4k

Reply:
Yeah, good call. I think you're right.